src/components/academies/AcademyDetailPage.jsx

```diff
@@ -64,12 +65,53 @@ const AcademyDetailPage = () => {
           />
         </div>
         <div>
-          <h2 data-testid="academy-title" className="mb-3 mt-3">{academy?.title}</h2>
-          <p data-testid="academy-description">{academy?.longDescription}</p>
+          <h1 data-testid="academy-title" className="mb-4 mt-3 font-italic text-left academy-title">
+            <FormattedMessage
+              id="academy.detail.page.academy.title"
+              defaultMessage="{academyTitle} Academy"
+              description="Title for the academy on the academy detail page"
+              values={{ academyTitle: academy?.title || 'Academy' }}
```

Comment:
nit: Maybe instead of `Academy` we should return  empty string in this expression?

```
values={{ academyTitle: academy?.title || '' }}
```

Reply:
@saleem-latif it would be a rare scenario but if academy title is not available we decided to show "Academy" as header..